Packs/HealthCheck/Scripts/HealthCheckAPIvalidation/HealthCheckAPIvalidation.py

```diff
@@ -32,18 +34,26 @@ def isAdminAPIInstance():
             },
         })
     for module in res:
-        if module['Contents']['response']['id'] == 'admin':
-            isAdminExist = True
+        if isinstance(module['Contents'], str):
+            return_error(module['Contents'])
+        elif module.get('Contents',{}).get('response',{}).get('defaultAdmin',{}) == True:
+            isDefaultAdminExist = True
+        else:
+            continue
 
-    return isAdminExist
+    return isDefaultAdminExist
 
 
 errors = []
-# Check if Demisto REST API integration was defined
-if not isDemistoAPIIntegrationAvailable():
+# Check if Demisto REST API integration was defined and number of instances
```

Comment:
You can add support for the `using` argument so the user can specify the specific integration instance he wants to use.
Otherwise, you are adding a pretty strict limitation for the script.

WDYT?

Reply:
how can I enforce customers to use `using` in ooo PB task?
this restriction is based on the fact that 2 instances provide completely wrong data.